### DISCUSSION TITLE
DreamD9 not accessible after days of non-use

### DISCUSSION THREAD

**Discussion by VossiVoss** [Hypfer/Valetudo#1685] [Q&A / Support] [answered]
My DreamD9 running Valetudo 2023.0.1 is no longer accessible after some days of non-use, neither via http nor via ssh. It seems that there is a sleep mode that is initiated after a certain period of inactivity. How to prevent this? I could create a routine with HomeAssistant that pings the robot or something similar. Would that help?

**Comment by Hypfer** [comment, marked as answer]
Is it docked while not in use? Does the dock have power?

**Comment by VossiVoss** [comment]
Yes, it's docked and on power. 

Edit: And the blue Wifi-LED is on.

**Reply by Hypfer** [reply]
The next release of Valetudo contains a mitigation against similar issues:
It pings its gateway every n minutes.

You might want to switch to nightly once to try that. It might help here

**Reply by VossiVoss** [reply]
Okay, I will try and report the results here. 
But is it possible to switch back from nigthly to the regular release later for a productive system?

**Reply by VossiVoss** [reply]
The update to the nightly build 2023-01-08T02:32:08.685Z runs completely through. But under "System Information" the old release number 2023.01.0 commit da42a6e8b3dcfe15f2693b5e6acde838b72059ad is still displayed. After that, the updater refers to the nightly 2023-01-08T02:32:08.685Z again, as if it is not installed yet. It seems to me that the update is not going through at all.

**Reply by Hypfer** [reply]
> After that, the updater refers to the nightly 2023-01-08T02:32:08.685Z again

That's normal and to be expected. The nightly update provider doesn't do any version checks but just pulls the repo.

The commit https://github.com/Hypfer/Valetudo/commit/da42a6e8b3dcfe15f2693b5e6acde838b72059ad that you've mentioned is a different commit then the one that was running before.

**Reply by VossiVoss** [reply]
Ah, thanks for the explanations, now I am a bit smarter ;-) 

I'll let you know if the update has fixed the error.

**Reply by VossiVoss** [reply]
Two weeks after applying the patch, I no longer have any problems with the robot's accessibility. The fix works. 
Great work and many thanks to the developers!

**Reply by Hypfer** [reply]
Thanks for reporting back!
